cypress/integration/rules.spec.js

```diff
@@ -67,6 +67,26 @@ describe('Rules page', () => {
         });
     });
 
+    it('creating a rule should add it to the list', () => {
+      cy.visit('/#/rules');
+      cy.get('a').contains('Add a rule').should('exist').as('addRule');
+      cy.get('@addRule').should('have.attr', 'href', '#/rules/add');
+      cy.get('@addRule').click();
+
+      cy.get('textarea').clear({force: true});
+      const newRule = '{{}"name":"newRule","workflow":"HelloWorldWorkflow","provider":"PODAAC_SWOT","collection":{{}"name":"MOD09GQ","version":"006"},"meta":{{}},"rule":{{}"type":"onetime","value":""},"state":"ENABLED"}';
+      cy.get('textarea').type(newRule, {force: true});
```

Comment:
You need to make sure to clear the existing contents of the textarea otherwise you can end up typing at the end of the existing JSON which gives an error when you submit. 

<img width="1440" alt="screen shot 2018-10-29 at 10 31 56 am" src="https://user-images.githubusercontent.com/1001694/47657158-ea5e5d00-db66-11e8-8e6f-452573ce7e12.png">


```suggestion
      cy.get('textarea')
        .type('{selectall}{backspace}`, { force: true })
        .type(newRule, {force: true});
```

Reply:
Hmmm the clear on line 76 was suppose to take care of that. It works for me like 80% of the time.

Reply:
Yeah, we're just running into the same general issue where we need a reliable way of updating the contents of these textareas

Reply:
Added a second clear makes it work in 100% of my test runs. Might seem silly but it gets up past the textarea issue right now